bundles/binding/org.openhab.binding.fritzboxtr064/src/main/java/org/openhab/binding/fritzboxtr064/internal/ItemMap.java

```diff
@@ -0,0 +1,99 @@
+package org.openhab.binding.fritzboxtr064.internal;
+
+/***
+ * Represents a item mapping. An item mapping is a collection of all parameters 
+ * based on the config string of the item. config string is like the key of an item mapping
+ * Item Mappings must be created manually to support desired fbox tr064 functions.
+ * 
+ * @author Boris Bock
+ *
+ */
+public class ItemMap{
```

Comment:
well, an instance of this class maps the item configuration from the *.items file to the TR064 parameters.